### DISCUSSION TITLE
`seasonal` bit on directed edge?

### DISCUSSION THREAD

**Discussion by chrstnbwnkl** [valhalla/valhalla#4944] [General]
The bit is set based on the "seasonal=*" tag but never used in the code base. It seems that without any further information on the exact time this tag applies, it seems of little use. Maybe a chance to reclaim it and use it for something else?

**Comment by kevinkreiser** [comment]
even if it was never used, if we set it to spare and then do use it, it means that old data would have a value that is based on the seasonal tag and not on whatever other thing we decide to use it for. this would mean old data would not have forward compatibility which is sadly something we do have to care about. 

i wish i had more time.. i think in general we should cut a v3 final branch and start calling master v4. then we can start edward scissor hands'ing the data over the course of a year or something. we could back port fixes to v3 for a time or allow others to do so if they are stuck on v3. but i think in general we should decide to move forward at some point. id like to maybe start a hit list of the thigns we want to do with the data format in the future. it would be a ton of work so im also a bit skeptical of feasibility to some degree which is why i would favor a model where we allow breakage in the compatibility layer for some time while v4 is in alpha/beta
